PortMaster/retrodeck/control.txt

```diff
@@ -1,16 +1,12 @@
 #!/bin/bash
 
-# Referenced by RetroDeck when the app is running. Might be needed at future point.
-#source /app/libexec/global.sh
-
-
-if [ -d "${HOME}/.var/app/net.retrodeck.retrodeck/config/retrodeck/" ]; then
-  controlfolder=$(grep 'roms_folder' "${HOME}/.var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg" | awk -F= '{print $2}')"/ports/PortMaster"
+if [ -d "/app/bin/retrodeck.sh" ]; then
+  source /app/libexec/global.sh # loading the RetroDECK framework that even give access to variables such as roms_folder
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+  directory="$roms_folder/portmaster"
```

Comment:
This means ports will live in: `$roms_folder/portmaster/ports/`, is that acceptable?

Reply:
Probably not optimal actually but might work for the moment. However I might find a way to tweak it later.
My idea is to have this structure:

$roms
- portmaster
 - PortMaster.sh/PortMaster.sh
 - game1.sh/game1.sh
 - game99.sh/game99.sh
 
In this way both the games and PortMaster will be recognized ad files a folders, and cloud be run with 1 button press, but I have understand how much this is possible.